lua/nvim-tree/actions/moves/item.lua

```diff
@@ -8,8 +8,9 @@ local M = {}
 
 ---@param where string
 ---@param what string
+---@param opts table|nil
 ---@return fun()
-function M.fn(where, what)
+function M.fn(where, what, opts)
```

Comment:
nil check please. Existing mappings pass nil and blow up :)

Reply:
I know, it was in fact temporary, with #2584 everything will work fine :)